fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBIndexOutput.java

```diff
@@ -157,13 +158,21 @@ public void writeByte(final byte b) {
         }
     }
 
+    void setNumberOfBytesToPrefetch(@Nonnull final PrefetchableBufferedChecksumIndexInput input, final long numBytes) {
```

Comment:
This might also be a little clearer if called `prefetch` or `prefetchBytes`

Reply:
It does not actually prefetch the bytes but keep a buffer of blocks based on the bytes expected to be read...

Reply:
attempted a rename to clarify.